src/index.tsx

```diff
@@ -2,14 +2,14 @@ import './index.css'
 import App from './App'
 import reportWebVitals from './reportWebVitals'
 import React from 'react'
-import * as ReactDOM from 'react-dom/client'
+// import * as ReactDOM from 'react-dom/client'
 import { Provider } from 'react-redux'
 import { persistStore } from 'redux-persist'
 import { PersistGate } from 'redux-persist/integration/react'
 import { store } from './state/store'
 import { render } from 'react-dom'
 
-const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement)
+// const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement)
```

Comment:
if we don't need this code, let's just remove it.

Reply:
> if we don't need this code, let's just remove it.

alright :)